src/storage/storageViewer.js

```diff
@@ -1,10 +1,15 @@
 'use strict'
 var helper = require('../helpers/util')
+var mappingPreimagesExtractor = require('./mappingPreimages')
 
 class StorageViewer {
   constructor (_context, _storageResolver, _traceManager) {
     this.context = _context
     this.storageResolver = _storageResolver
+    // contains [mappingSlot][mappingkey] = preimage
+    // this map is renewed for each execution step
```

Comment:
Should only do it once at the beginning of the debugging session and use the trace otherwise.